.github/workflows/testing.yml

```diff
@@ -16,6 +16,9 @@ permissions:
   contents: read
 
 jobs:
+  lint:
+    uses: stylelint/.github/.github/workflows/lint.yml@reusable-workflow # TODO: Change to `@main`
+
   test:
     name: Test on Node.js ${{ matrix.node }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
```

Comment:
[note] It's difficult to use `stylelint/.github/.github/workflows/lint.yml` for the current `test` job because it has more complex steps to report coverage.